### ISSUE TITLE
Update README with the _formatted issue

### ISSUE THREAD

**Issue by juchom** [meilisearch/meilisearch-dotnet#392]
It would be a good idea to give information to users that want to use the highlighting feature or Meilisearch that it is not available out of the box right now.

This would avoid them to waste too much time and point them to #315 for a solution and track this issue.

**Comment by alallema** [comment]
Yes, thank you @juchom for the suggestion it would indeed be a very good idea.

**Comment by juchom** [comment]
The credit goes to @Christoph-Wagner :)

**Comment by ahmednfwela** [comment]
would this issue still be needed if https://github.com/meilisearch/meilisearch-dotnet/pull/406 lands ?
